src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,9 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User testUser = userService.findByEmail(email, password);
+        return testUser != null && password.equals(testUser.getPassword()) ? true : false;
```

Comment:
```suggestion
        User user = userService.findByEmail(email, password);
        return testUser != null && password.equals(user.getPassword())
```
Let's simplify your code